### PR TITLE
Timer publisher

### PR DESCRIPTION
**Pull request by filipkauffeldt**
Add abstract classes for handling time and subscribing to events that update lights

Should look something like this
image

### CODE REVIEW

**Review by RezaAmanii — APPROVED** [review]
Approved

File: abstract_light_service.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from abc import ABC, abstractmethod
+from model.events.lamp_event import LampEvent
+
+class AbstractLightService(ABC):
+    def __init__(self) -> None:
+        pass
+
+    @abstractmethod
+    def handleEvent(self, event: LampEvent) -> None:
```
Comment (Flaskhals68) [inline abstract_light_service.py]: camel casing?

Comment (filipkauffeldt) [reply abstract_light_service.py]: Fixed

**Review by BooloCaptain — APPROVED** [review]
gangstruös pr b

**Pull request merged**